api-auto-registration/usage.hbs.md

```diff
@@ -83,75 +99,44 @@ spec:
 Once the supply chain runs it will create an `APIDescriptor` custom resource. This resource is what TAP uses to auto register your API. See below
 for more details.
 
-## <a id='without-supply-chain'></a>Without the OOTB Supply Chains
- If you are not using supply chains, or are creating custom supply chains you can still utilize API Auto Registration. You will need to directly create an APIDescriptor custom resource.  See the examples below for further details.
+## <a id='using-custom-supply-chain'></a>Using Custom Supply Chains
+If you are creating custom supply chains, you can still utilize API Auto Registration. To write your own Supply Chain pipeline, 
+you can use `ClusterConfigTemplate` by the name of `config-template` in your pipeline. To write your own custom task, 
+you can check how the template is written to read params, interpret baseURL from Knative Services and construct APIDescriptor CRs.
 
-## <a id='api-descriptor'></a>APIDescriptor Explained
+In the Delivery pipeline, you will need to directly create an APIDescriptor custom resource. You may need to grant some permissions to create the CR from the Delivery pipeline. 
+For more info on APIDescriptors, check out the [Key Concepts section](key-concepts.md)
 
-For API Auto Registration to take place a custom resource of type `APIDescriptor` needs to be created. If you use the out OOTB supply chains and have set the required fields on your workload one will automatically be created for you, otherwise you will need to make one yourself.
-Below are the details of this custom resource.
+## <a id='using-gitops-manually'></a>Using other GitOps processes or Manually
+Using your GitOps process (or manually), you will need to stamp out an APIDescriptor CR and apply it in the desired cluster. 
+The APIDescriptor will need all the required fields to successfully reconcile.
 
-### <a id='absolute-url'></a>With an Absolute URL
+For more info on APIDescriptors, check out the [Key Concepts section](key-concepts.md)
 
-```yaml
-apiVersion: apis.apps.tanzu.vmware.com/v1alpha1
-kind: APIDescriptor
-metadata:
-  name: sample-absolute-url
-spec:
-  type: openapi
-  description: A set of API endpoints to manage the resources within the petclinic app.
-  system: spring-petclinic
-  owner: team-petclinic
-  location:
-    path: "/v3/api-docs.yaml"
-    baseURL:
-      url: https://myservice.com
-```
-
-### <a id='with-ref'></a>With a Ref
+## <a id='update-values'></a>Update install values for api-auto-registration package
```

Comment:
This section is almost exactly the same as the install guide. Could we just point them there?

Another option for updating install value is to update tap-values.yaml then update TAP. Not sure how other teams document that, maybe worth following the same pattern.